### PR TITLE
Branch add and filter error message changes

### PR DESCRIPTION
**Pull request by knotstoks**
Changed error messages for add and filter according to the degen 4am meeting!

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -275,21 +275,21 @@ Example:<br>
 
 Filters tutor(s) from the list of matched tutors.
 
-Format: <br>`filter t [n/NAME] [g/GENDER] [q/QUALIFICATION]`
+Format: <br>`filter [n/NAME] [g/GENDER] [q/QUALIFICATION]`
```
Comment (tenebrius1) [inline docs/UserGuide.md:278]: I think Xin Yan edited this part in her branch also? Can you confirm? @Lemonsr

File: src/main/java/seedu/address/logic/parser/AddCommandParser.java [hunk 2]
```diff
@@ -54,32 +55,45 @@ public AddCommand parse(String args) throws ParseException {
...
             } else {
                 remark = ParserUtil.parseRemark(argMultimap.getValue(PREFIX_REMARK).get());
             }
+
             Set<Tag> tagList = ParserUtil.parseTags(argMultimap.getAllValues(PREFIX_TAG));
 
             Tutor tutor = new Tutor(name, phone, gender, qualification, remark, tagList);
             return new AddCommand(tutor, PersonType.TUTOR);
             // No break necessary due to return statement
         case STUDENT:
```
Comment (tenebrius1) [inline src/main/java/seedu/address/logic/parser/AddCommandParser.java:79]: The parse method is over 30 lines long, you might want to abstract out some parts so you can have better SLAP?

File: docs/UserGuide.md (hunk 1 above)
Comment (Lemonsr) [reply docs/UserGuide.md:278]: Yep I did!

File: src/main/java/seedu/address/logic/parser/AddCommandParser.java (hunk 2 above)
Comment (Lemonsr) [reply src/main/java/seedu/address/logic/parser/AddCommandParser.java:79]: Actually I think the 30 lines long thing has nothing to do with SLAP..? 😂 SLAP is about the same level of abstraction within a code fragment, but shortening the method length is good for readability~

File: src/main/java/seedu/address/logic/commands/AddCommand.java [hunk 3]
```diff
@@ -23,27 +23,27 @@
...
-            + " or <t "
+            + PREFIX_TAG + "TAG...>\n"
+            + "or <s "
             + PREFIX_NAME + "NAME "
             + PREFIX_PHONE + "PHONE "
             + PREFIX_GENDER + "GENDER "
-            + PREFIX_QUALIFICATION + "QUALIFICATION "
             + "[" + PREFIX_REMARK + "REMARK] "
-            + PREFIX_TAG + "TAG...>\n"
+            + PREFIX_TAG + "TAG>\n"
```
Comment (kleonang) [inline src/main/java/seedu/address/logic/commands/AddCommand.java:40]: Students can also have multiple tags:
```suggestion
            + PREFIX_TAG + "TAG...>\n"
```

File: src/main/java/seedu/address/model/person/Gender.java [hunk 4]
```diff
@@ -8,7 +8,7 @@
  * Guarantees: immutable; is valid as declared in {@link #isValidGender(String)}
  */
 public class Gender {
-    public static final String MESSAGE_CONSTRAINTS = "Gender should be 'M' or 'F'";
+    public static final String MESSAGE_CONSTRAINTS = "Gender should be 'M/m' or 'F/f'";
```
Comment (kleonang) [inline src/main/java/seedu/address/model/person/Gender.java:11]: Maybe clearer to indicate the part in quotes as literal input:
```suggestion
    public static final String MESSAGE_CONSTRAINTS = "Gender should be 'M'/'m' or 'F'/'f'";
```

**Review by kleonang — COMMENTED** [review]
Minor syntax comments

**Review by tenebrius1 — APPROVED** [review]
LGTM! 🚀

**Pull request merged**